src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "3.13.6",
+  "version": "3.14.6",
```

Comment:
- [ ] New version should be `3.14.0`

Reply:
3.23.4 , right? 

Is it okey to merge this now?